edit/edit.js

```diff
@@ -259,6 +259,13 @@ function initStyleData() {
       }
       return style;
     });
+
+  function fetchStyle() {
```

Comment:
Doesn't this need an `id` parameter?

Reply:
It references the `id` variable defined at L231.

Reply:
Oh okay, it threw me off since I was only looking at a snippet of the whole function, and `fetchStyle` is defined after the return.